<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPPNetworkClientClass</class>
 <widget class="QMainWindow" name="CPPNetworkClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPPNetworkClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="celsiusButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Récupérer la température en degrés Celsius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fahrenheitButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Récupérer la température en degrés Fahrenheit </string>
    </property>
   </widget>
   <widget class="QPushButton" name="hygometrieButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Récupérer la valeur de l’hygrométrie </string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="portStatusLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satut connexion : déconnectee</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisir un capteur</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CPPNetworkClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>CPPNetworkClientClass</receiver>
   <slot>connectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>celsiusButton</sender>
   <signal>clicked()</signal>
   <receiver>CPPNetworkClientClass</receiver>
   <slot>celsiusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hygometrieButton</sender>
   <signal>clicked()</signal>
   <receiver>CPPNetworkClientClass</receiver>
   <slot>hygometrieButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fahrenheitButton</sender>
   <signal>clicked()</signal>
   <receiver>CPPNetworkClientClass</receiver>
   <slot>farhenheitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectButtonClicked()</slot>
  <slot>celsiusButtonClicked()</slot>
  <slot>farhenheitButtonClicked()</slot>
  <slot>hygometrieButtonClicked()</slot>
 </slots>
</ui>
